scripts/py_matter_yamltests/matter_yamltests/parser.py

```diff
@@ -845,6 +845,7 @@ def __init__(self, test_file, pics_file, definitions):
 
         pics_checker = PICSChecker(pics_file)
         tests = _value_or_none(data, 'tests')
+        self.is_test_pics_enabled = pics_checker.check(self.PICS)
```

Comment:
Please see https://github.com/project-chip/connectedhomeip/pull/24741#issuecomment-1409268185

Reply:
@vivien-apple 

I am confused by this top level PICS then, what are we supposed to compare it to?

Reply:
I think in general we do want to use this top level PICS. But...
I believe this top level PICS code has been added by the Test Harness team and our CI has not been updated to take it into account :/

For example, looking at `Test_TC_CC_1_1` you can see a top level PICS code: 
```
PICS:
    -  CC.S
```

Now looking into the default PICS code for ci in `src/app/tests/suites/certification/ci-pics-values` there is no such PICS code.

So if you filter based on this, running `Test_TC_CC_1_1` will go from running:
```
✓ 82.0ms	Parsing: src/app/tests/suites/certification/Test_TC_CC_1_1.yaml
Running: "25.1.1. [TC-CC-1.1] Global attributes with server as DUT" with 47 tests
		1. Running Wait for the commissioned device to be retrieved
		2. Running read the global attribute: ClusterRevision
		̶̶̶̶̶3̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶g̶l̶o̶b̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶:̶ ̶F̶e̶a̶t̶u̶r̶e̶M̶a̶p
		̶̶̶̶̶4̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶G̶i̶v̶e̶n̶ ̶C̶C̶.̶S̶.̶F̶0̶0̶(̶H̶S̶)̶ ̶e̶n̶s̶u̶r̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶m̶a̶p̶ ̶h̶a̶s̶ ̶t̶h̶e̶ ̶c̶o̶r̶r̶e̶c̶t̶ ̶b̶i̶t̶ ̶s̶e̶t
		5. Running Given CC.S.F01(EHue) ensure featuremap has the correct bit set
		6. Running Given CC.S.F02(CL) ensure featuremap has the correct bit set
		7. Running Given CC.S.F03(XY) ensure featuremap has the correct bit set
		8. Running Given CC.S.F04(CT) ensure featuremap has the correct bit set
		9. Running Read the global attribute: AttributeList
		10. Running Read the optional attribute(CurrentHue) in AttributeList
		11. Running Read the optional attribute(CurrentSaturation) in AttributeList
		12. Running Read the optional attribute(RemainingTime) in AttributeList
		13. Running Read the optional attribute(CurrentX) in AttributeList
		14. Running Read the optional attribute(CurrentY) in AttributeList
		15. Running Read the optional attribute(DriftCompensation) in AttributeList
		16. Running Read the optional attribute(CompensationText) in AttributeList
		17. Running Read the optional attribute(ColorTemperatureMireds) in AttributeList
		18. Running Read the optional attribute(EnhancedCurrentHue) in AttributeList
		19. Running Read the optional attribute(ColorLoopActive) in AttributeList
		20. Running Read the optional attribute(ColorLoopDirection) in AttributeList
		21. Running Read the optional attribute(ColorLoopTime) in AttributeList
		22. Running Read the optional attribute(ColorLoopStartEnhancedHue) in AttributeList
		23. Running Read the optional attribute(ColorLoopStoredEnhancedHue) in AttributeList
		24. Running Read the optional attribute(ColorTempPhysicalMinMireds) in AttributeList
		25. Running Read the optional attribute(ColorTempPhysicalMaxMireds) in AttributeList
		26. Running Read the optional attribute(CoupleColorTempToLevelMinMireds) in AttributeList
		27. Running Read the optional attribute(StartUpColorTemperatureMireds) in AttributeList
		28. Running Read the optional command(MoveToHue) in AcceptedCommandList
		29. Running Read the optional command(MoveHue) in AcceptedCommandList
		30. Running Read the optional command(StepHue) in AcceptedCommandList
		31. Running Read the optional command(MoveToSaturation) in AcceptedCommandList
		32. Running Read the optional command(MoveSaturation) in AcceptedCommandList
		33. Running Read the optional command(StepSaturation) in AcceptedCommandList
		34. Running Read the optional command(MoveToHueAndSaturation) in AcceptedCommandList
		35. Running Read the optional command(MoveToColor) in AcceptedCommandList
		36. Running Read the optional command(MoveColor) in AcceptedCommandList
		37. Running Read the optional command(StepColor) in AcceptedCommandList
		38. Running Read the optional command(MoveToColorTemperature) in AcceptedCommandList
		39. Running Read the optional command(EnhancedMoveToHue) in AcceptedCommandList
		40. Running Read the optional command(EnhancedMoveHue) in AcceptedCommandList
		41. Running Read the optional command(EnhancedStepHue) in AcceptedCommandList
		42. Running Read the optional command(EnhancedMoveToHueAndSaturation) in AcceptedCommandList
		43. Running Read the optional command(ColorLoopSet) in AcceptedCommandList
		44. Running Read the optional command(StopMoveStep) in AcceptedCommandList
		45. Running Read the optional command(MoveColorTemperature) in AcceptedCommandList
		46. Running Read the optional command(StepColorTemperature) in AcceptedCommandList
		47. Running Read the global attribute: GeneratedCommandList

```

to this:
```
✓ 64.0ms	Parsing: src/app/tests/suites/certification/Test_TC_CC_1_1.yaml
Running: "25.1.1. [TC-CC-1.1] Global attributes with server as DUT" with 47 tests
		̶̶̶̶̶1̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶W̶a̶i̶t̶ ̶f̶o̶r̶ ̶t̶h̶e̶ ̶c̶o̶m̶m̶i̶s̶s̶i̶o̶n̶e̶d̶ ̶d̶e̶v̶i̶c̶e̶ ̶t̶o̶ ̶b̶e̶ ̶r̶e̶t̶r̶i̶e̶v̶e̶d
		̶̶̶̶̶2̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶r̶e̶a̶d̶ ̶t̶h̶e̶ ̶g̶l̶o̶b̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶:̶ ̶C̶l̶u̶s̶t̶e̶r̶R̶e̶v̶i̶s̶i̶o̶n
		̶̶̶̶̶3̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶g̶l̶o̶b̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶:̶ ̶F̶e̶a̶t̶u̶r̶e̶M̶a̶p
		̶̶̶̶̶4̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶G̶i̶v̶e̶n̶ ̶C̶C̶.̶S̶.̶F̶0̶0̶(̶H̶S̶)̶ ̶e̶n̶s̶u̶r̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶m̶a̶p̶ ̶h̶a̶s̶ ̶t̶h̶e̶ ̶c̶o̶r̶r̶e̶c̶t̶ ̶b̶i̶t̶ ̶s̶e̶t
		̶̶̶̶̶5̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶G̶i̶v̶e̶n̶ ̶C̶C̶.̶S̶.̶F̶0̶1̶(̶E̶H̶u̶e̶)̶ ̶e̶n̶s̶u̶r̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶m̶a̶p̶ ̶h̶a̶s̶ ̶t̶h̶e̶ ̶c̶o̶r̶r̶e̶c̶t̶ ̶b̶i̶t̶ ̶s̶e̶t
		̶̶̶̶̶6̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶G̶i̶v̶e̶n̶ ̶C̶C̶.̶S̶.̶F̶0̶2̶(̶C̶L̶)̶ ̶e̶n̶s̶u̶r̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶m̶a̶p̶ ̶h̶a̶s̶ ̶t̶h̶e̶ ̶c̶o̶r̶r̶e̶c̶t̶ ̶b̶i̶t̶ ̶s̶e̶t
		̶̶̶̶̶7̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶G̶i̶v̶e̶n̶ ̶C̶C̶.̶S̶.̶F̶0̶3̶(̶X̶Y̶)̶ ̶e̶n̶s̶u̶r̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶m̶a̶p̶ ̶h̶a̶s̶ ̶t̶h̶e̶ ̶c̶o̶r̶r̶e̶c̶t̶ ̶b̶i̶t̶ ̶s̶e̶t
		̶̶̶̶̶8̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶G̶i̶v̶e̶n̶ ̶C̶C̶.̶S̶.̶F̶0̶4̶(̶C̶T̶)̶ ̶e̶n̶s̶u̶r̶e̶ ̶f̶e̶a̶t̶u̶r̶e̶m̶a̶p̶ ̶h̶a̶s̶ ̶t̶h̶e̶ ̶c̶o̶r̶r̶e̶c̶t̶ ̶b̶i̶t̶ ̶s̶e̶t
		̶̶̶̶̶9̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶g̶l̶o̶b̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶:̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶0̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶u̶r̶r̶e̶n̶t̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶1̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶u̶r̶r̶e̶n̶t̶S̶a̶t̶u̶r̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶2̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶R̶e̶m̶a̶i̶n̶i̶n̶g̶T̶i̶m̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶3̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶u̶r̶r̶e̶n̶t̶X̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶4̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶u̶r̶r̶e̶n̶t̶Y̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶5̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶D̶r̶i̶f̶t̶C̶o̶m̶p̶e̶n̶s̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶6̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶m̶p̶e̶n̶s̶a̶t̶i̶o̶n̶T̶e̶x̶t̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶7̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶T̶e̶m̶p̶e̶r̶a̶t̶u̶r̶e̶M̶i̶r̶e̶d̶s̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶8̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶E̶n̶h̶a̶n̶c̶e̶d̶C̶u̶r̶r̶e̶n̶t̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶1̶9̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶L̶o̶o̶p̶A̶c̶t̶i̶v̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶0̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶L̶o̶o̶p̶D̶i̶r̶e̶c̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶1̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶L̶o̶o̶p̶T̶i̶m̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶2̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶L̶o̶o̶p̶S̶t̶a̶r̶t̶E̶n̶h̶a̶n̶c̶e̶d̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶3̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶L̶o̶o̶p̶S̶t̶o̶r̶e̶d̶E̶n̶h̶a̶n̶c̶e̶d̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶4̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶T̶e̶m̶p̶P̶h̶y̶s̶i̶c̶a̶l̶M̶i̶n̶M̶i̶r̶e̶d̶s̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶5̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶l̶o̶r̶T̶e̶m̶p̶P̶h̶y̶s̶i̶c̶a̶l̶M̶a̶x̶M̶i̶r̶e̶d̶s̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶6̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶C̶o̶u̶p̶l̶e̶C̶o̶l̶o̶r̶T̶e̶m̶p̶T̶o̶L̶e̶v̶e̶l̶M̶i̶n̶M̶i̶r̶e̶d̶s̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶7̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶(̶S̶t̶a̶r̶t̶U̶p̶C̶o̶l̶o̶r̶T̶e̶m̶p̶e̶r̶a̶t̶u̶r̶e̶M̶i̶r̶e̶d̶s̶)̶ ̶i̶n̶ ̶A̶t̶t̶r̶i̶b̶u̶t̶e̶L̶i̶s̶t
		̶̶̶̶̶2̶8̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶T̶o̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶2̶9̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶0̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶S̶t̶e̶p̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶1̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶T̶o̶S̶a̶t̶u̶r̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶2̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶S̶a̶t̶u̶r̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶3̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶S̶t̶e̶p̶S̶a̶t̶u̶r̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶4̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶T̶o̶H̶u̶e̶A̶n̶d̶S̶a̶t̶u̶r̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶5̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶T̶o̶C̶o̶l̶o̶r̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶6̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶C̶o̶l̶o̶r̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶7̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶S̶t̶e̶p̶C̶o̶l̶o̶r̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶8̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶T̶o̶C̶o̶l̶o̶r̶T̶e̶m̶p̶e̶r̶a̶t̶u̶r̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶3̶9̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶E̶n̶h̶a̶n̶c̶e̶d̶M̶o̶v̶e̶T̶o̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶0̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶E̶n̶h̶a̶n̶c̶e̶d̶M̶o̶v̶e̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶1̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶E̶n̶h̶a̶n̶c̶e̶d̶S̶t̶e̶p̶H̶u̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶2̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶E̶n̶h̶a̶n̶c̶e̶d̶M̶o̶v̶e̶T̶o̶H̶u̶e̶A̶n̶d̶S̶a̶t̶u̶r̶a̶t̶i̶o̶n̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶3̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶C̶o̶l̶o̶r̶L̶o̶o̶p̶S̶e̶t̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶4̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶S̶t̶o̶p̶M̶o̶v̶e̶S̶t̶e̶p̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶5̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶M̶o̶v̶e̶C̶o̶l̶o̶r̶T̶e̶m̶p̶e̶r̶a̶t̶u̶r̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶6̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶o̶p̶t̶i̶o̶n̶a̶l̶ ̶c̶o̶m̶m̶a̶n̶d̶(̶S̶t̶e̶p̶C̶o̶l̶o̶r̶T̶e̶m̶p̶e̶r̶a̶t̶u̶r̶e̶)̶ ̶i̶n̶ ̶A̶c̶c̶e̶p̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
		̶̶̶̶̶4̶7̶.̶ ̶R̶u̶n̶n̶i̶n̶g̶ ̶R̶e̶a̶d̶ ̶t̶h̶e̶ ̶g̶l̶o̶b̶a̶l̶ ̶a̶t̶t̶r̶i̶b̶u̶t̶e̶:̶ ̶G̶e̶n̶e̶r̶a̶t̶e̶d̶C̶o̶m̶m̶a̶n̶d̶L̶i̶s̶t
```

So reading the PICS code is not bad in itself, and at the end, we do want to use it I guess, it would allow us to skip things that are not runned in CI beside setup/teardown steps (there are a few occurrences of such things in the color control tests).
But at the moment you can not filter based on it, because we need to manually add the values to `src/app/tests/suites/certification/ci-pics-values` and take into account that this top level PICS things is a list (I have no idea why at the moment...)